executor/analyze.go

```diff
@@ -1450,9 +1528,12 @@ workLoop:
 				resultCh <- err
 				continue
 			}
+			finalMemSize := hist.MemoryUsage() + topn.MemoryUsage()
+			e.memTracker.Consume(finalMemSize)
 			hists[task.slicePos] = hist
 			topns[task.slicePos] = topn
 			resultCh <- nil
+			e.memTracker.Consume(-totalMemInc)
```

Comment:
I think we only need to add `hist.MemoryUsage() + topn.MemoryUsage()`

Reply:
the intermediate memory size is not small enough to ignore.